lang/po/new/cs.po

```diff
@@ -858,7 +855,7 @@ msgstr "Zadna SD karta"
 # MSG_NA c=3
 #: messages.c:127
 msgid "N/A"
-msgstr ""
+msgstr "N/A"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.